cg/meta/workflow/analysis.py

```diff
@@ -75,7 +75,7 @@ def check_analysis_ongoing(self, case_id: str) -> None:
     def verify_case_path_exists(self, case_id: str) -> None:
         if not self.get_case_path(case_id=case_id).exists():
             LOG.error("Working directory path for %s does not exist", case_id)
-            raise CgError()
+            raise CgError("No working directory for %s exists", case_id)
```

Comment:
On second thought
```suggestion
            raise CgError(f"No working directory for {case_id} exists")
```

Reply:
Should probably apply this to the line above as well then 👍

Reply:
Updated both in 3a22bdf5